conf/distro/include/arm-defaults.inc

```diff
@@ -0,0 +1,31 @@
+# This function changes the default tune for machines which
+# are based on armv7a to use common tune value, note that we enforce hard-float
+# which is default on Ångström for armv7+
```

Comment:
No strong opinion on that, trying to keep the diff as small as possible, but changing the comment won't hurt merging code changes between rpb and angstrom.